src/Views/Menu.java

```diff
@@ -71,9 +130,7 @@ public void printMainMenu() {
             try {
                 selection = Integer.parseInt(scanner.nextLine());
                 validInput = true;
-            } catch (InputMismatchException e) {
-                System.out.println("What you entered was not a number.");
-            } catch (NumberFormatException e) {
+            } catch (InputMismatchException | NumberFormatException e) {
```

Comment:
I had to add the NumberFormatException catch as well because without it if the user just hits enter at the prompt it throws that one instead of InputMismatchException.  I think we need both.